addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -139,6 +139,13 @@ $-editor-messages-margin-x: 2%;
 [draggable] {
     user-select: none;
 }
+/* Prevent the native drag and drop for images. */
+img {
+    -webkit-user-drag: none !important;
+    -khtml-user-drag: none !important;
+    -moz-user-drag: none !important;
+    -o-user-drag: none !important;
+}
```

Comment:
How can I test the difference with or without this CSS in the editor ?

Reply:
Random comment: note that if the vendor prefixes are required, you should rather add them there: https://github.com/odoo/odoo/blob/master/odoo/addons/base/models/assetsbundle.py#L701